Projects/UsertInterfaces/DesignSystem/Sources/TextField/TookTextField.swift

```diff
@@ -0,0 +1,120 @@
+import SwiftUI
+
+public struct TookTextField: View {
+    var labelText: String
+    var placeholderText: String
+    var isError: Bool
+    var buttonAction: () -> Void
+    var onCommit: () -> Void
+    var errorText: String
+    @Binding var text: String
+    @FocusState var isFocus: Bool
+
+    public init(
+        _ textFieldHeader: String = "",
+        placeHolderText: String = "",
+        isError: Bool = false,
+        errorText: String = "",
+        text: Binding<String>,
+        buttonAction: @escaping () -> Void = {},
+        onCommit: @escaping () -> Void = {}
+    ) {
+        self.labelText = textFieldHeader
+        self.placeholderText = placeHolderText
+        self.isError = isError
+        self.errorText = errorText
+        self.buttonAction = buttonAction
+        self.onCommit = onCommit
+        self._text = text
+    }
```

Comment:
```suggestion
    public init(
        _ textFieldHeader: String = "",
        placeholderText: String = "",
        isError: Bool = false,
        errorText: String = "",
        text: Binding<String>,
        buttonAction: @escaping () -> Void = {},
        onCommit: @escaping () -> Void = {}
    ) {
        self.labelText = textFieldHeader
        self.placeholderText = placeholderText
        self.isError = isError
        self.errorText = errorText
        self.buttonAction = buttonAction
        self.onCommit = onCommit
        self._text = text
    }
```
placeHolder -> placeholder 수정합니다